platform/binance/model.go

```diff
@@ -147,6 +148,19 @@ func (a TokenPage) findToken(symbol string) *Token {
 	return nil
 }
 
+func (balance *Balance) isZeroBalance() bool {
+	switch {
+	case isNotZeroValue(balance.Free):
+		return false
+	case isNotZeroValue(balance.Frozen):
```

Comment:
maybe doing a loop instead?

Reply:
and call `valueFloat, err := strconv.ParseFloat(value, 64)`

Reply:
done